doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -73,10 +73,10 @@ This chapter presents a consolidated set of OpenStack Service APIs corresponding
 
 
 <a name="5.3"></a>
-## 5.3. Consolidated Set of APIs.
+## 5.3. Consolidated Set of APIs
 
 ### 5.3.1. OpenStack Interfaces
-This section illustrates some of the Interfaces provided by OpenStack; the exhaustive list of APIs is available at https://docs.openstack.org/api-ref/ and CNTT standard is to use the latest version v3. OpenStack REST APIs are simple to interact with using either of two options. Clients can either call the APIs directly using the HTTP or REST library, or they can use one of the many programming language specific cloud libraries. 
+This section illustrates some of the Interfaces provided by OpenStack; the exhaustive list of APIs is available at https://docs.openstack.org/api-ref/ and CNTT standard is to use the latest version of every OpenStack service. OpenStack REST APIs are simple to interact with using either of two options. Clients can either call the APIs directly using the HTTP or REST library, or they can use one of the many programming language specific cloud libraries. 
```

Comment:
I do not think that we should mandate the latest version of every OpenStack service. This would mean Train currently. We should either set a minimum OpenStack release (however not all projects do follow the release strategy) a component version for every service or an API version for every API.

Reply:
> CNTT standard is to use the latest version of every OpenStack service.

This chapter treats `Pike` as baseline and already specifies the minimum Openstack API/CLI version for most of the APIs. If not the **latest** one, what do we say as CNTT Standard? 
1. Do we specify a max supported version (current version?) along with minimum supported one?
1. We can delete the complete line about 'CNTT standard' and only keeps the baseline mentioning. 
1. All the releases of OpenStack (>= Pike) that gets certified by RC can be considered as a CNTT RA Compliant and can be used as CNTT standard.

Reply:
I would vote for 3.

Reply:
i would also vote for 3.

Reply:
Vote for 3 with the operative phrase being "get certified by RC"; implication if only Pike gets certified then Pike is CNTT RA Compliant. IMHO, we can only have one "CNTT standard."

May I suggest that if more than one OpenStack release gets certified then the "CNTT standard" be the minimal CNTT RA compliant OpenStack release.

Reply:
@pgoyal01 What is minimal? Which satisfies the **musts**? If yes, then it can be equally applicable to any two openstack releases.

Reply:
@deebhatia Thanks. Change from "minimal" to "oldest." 

New wording: When more than one OpenStack release gets certified then the "CNTT standard" will be the oldest CNTT RA compliant OpenStack release. Example, Let us assume that the Pike, Queens and Stein OpenStack releases get certified. Then the CNTT standard OpenStack version will be Pike.

Reply:
I will vote for 3 also.
We consider only OpenStack releases greater or equal to Pike release.
On this basis, we could rephrase the sentence introducing the APIs versions table by
 "The minimal versions of OpenStack services APIs are related to Pike release"

Reply:
@karinesevilla One possible solution from mentioning the OpenStack release within the Chapters is to use the "CNTT Standard OpenStack version" as suggested earlier. Now, we only have to find a suitable place, maybe in RA-1 ReadMe to define whatever is the applicable CNTT Standard OpenStack version is. 

W.r.t. API versions, we can then refer to API versions compatible with the "CNTT Standard OpenStack version."

Critique?

Reply:
@pgoyal01 Okay. You want to replace the term of `Baseline OpenStack Release Version` with `CNTT Standard OpenStack Version`. 
We already define Pike as baseline in this chapter's [introduction](https://cntt-n.github.io/CNTT/doc/ref_arch/openstack/chapters/chapter05.html#5.1). Should I move this statement there if the RA-1 readme movement is still debatable?

Reply:
@CsatariGergely Statement 3 added.

Reply:
is that resolved now?

Reply:
Resolved. :) @rabi-abdel Ideally, reviewer should resolve it.

Reply:
OK @CsatariGergely requested the change, can you @CsatariGergely resolve if this has been addressed please?